mod-di-converter-storage-server/src/main/java/org/folio/services/AbstractProfileService.java

```diff
@@ -198,14 +198,11 @@ private Future<Void> fillActionProfileDetailWrapper(ProfileAssociation actionPro
     String actionProfileId = actionProfileAssociation.getDetailProfileId();
     return profileWrapperDao.getWrapperByProfileId(actionProfileId, ACTION_PROFILE, tenantId)
       .compose(profileWrappers -> {
-        if (profileWrappers.size() == 1) {
-          actionProfileAssociation.setDetailWrapperId(profileWrappers.get(0).getId());
-          return Future.succeededFuture();
-        }
-        if (profileWrappers.size() > 1) {
-          return Future.failedFuture(new IllegalStateException(format("Found several wrappers for Action Profile with id '%s' ", actionProfileId)));
+        if (profileWrappers.size() == 0) {
+          return Future.failedFuture(new NotFoundException(format("Mapping Profile does NOT exist for this Action Profile with id '%s' ", actionProfileId)));
```

Comment:
Should the log say that the wrapper is missing? not the mapping profile

Reply:
Thanks! Fixed.